src/Containers/App/App.js

```diff
@@ -17,29 +16,29 @@ class App extends Component {
     const { notes, loading, togglePopup } = this.props
     return (
       <div>
-        {loading && <h1>Loading notes...</h1>}
-        {!loading && (
-          <Fragment>
-            <Header togglePopup={togglePopup}/>
-            <Switch>
-              <Route exact path="/" component={NotesSection} />
-              {/* <Route component={NotFound} /> */}
-            </Switch>
-            <Route path="/new-note" render={() => <NoteForm />} />
-            <Route
-              path="/notes/:id"
-              render={({ match }) => {
-                const { id } = match.params
-                const note = notes.find((note) => note.id === parseInt(id))
-                if (note) {
-                  return <NoteForm {...note} />
-                } else {
-                  return <NotFound />
-                }
-              }}
-            />
-          </Fragment>
-        )}
+        <Fragment>
```

Comment:
Can you explain why you used <Fragment> here?
Otherwise, everything else looks great :)